harmonica/tests/test_transformations.py

```diff
@@ -258,7 +258,7 @@ def test_derivative_upward(sample_potential, sample_g_z):
         pad_width=pad_width,
     )
     # Calculate upward derivative and unpad it
-    derivative = derivative_upward(potential_padded)
+    derivative = -derivative_upward(potential_padded)
```

Comment:
This is the wrong place for the `-`. It should be further down in line 266. There we have:

```python
g_z = sample_g_z[trim:-trim, trim:-trim] * 1e-5  # convert to SI units
```

but it should be:

```python
g_up = -sample_g_z[trim:-trim, trim:-trim] * 1e-5  # convert to SI units
```